### PR TITLE
Refactoring from meeting

### PR DESCRIPTION
**Pull request by VJubert**

### CODE REVIEW

File: src/Xmf2.Commons.iOS/Controls/NavBarWithTwoActions.cs [hunk 1]
```diff
@@ -19,49 +19,49 @@ public class NavBarWithTwoActions : UIView
 		public Action RightAction { get; set; }
 		public ICommand LeftCommand
 		{
-			get { return null; }
+			get => null;
 			set { LeftAction = () => value?.TryExecute(); }
```
Comment (Julien-Mialon) [inline src/Xmf2.Commons.iOS/Controls/NavBarWithTwoActions.cs]: use a lambda set ?

File: src/Xmf2.Commons.iOS/Controls/NavBarWithTwoActions.cs [hunk 2]
```diff
@@ -19,49 +19,49 @@ public class NavBarWithTwoActions : UIView
...
 		}
 		public ICommand RightCommand
 		{
-			get { return null; }
+			get => null;
 			set { RightAction = () => value?.TryExecute(); }
```
Comment (Julien-Mialon) [inline src/Xmf2.Commons.iOS/Controls/NavBarWithTwoActions.cs]: use a lambda set ?

File: src/Xmf2.Commons.iOS/Controls/UITextViewWithPlaceholder.cs [hunk 3]
```diff
@@ -41,8 +42,12 @@ public override string Text
 				this.UpdatePlaceHolderVisibility();
 			}
 		}
-		public UIColor PlaceholderColor { get { return _placeholderLabel.TextColor; } set { _placeholderLabel.TextColor = value; } }
-		public UIFont PlaceholderFont { get { return _placeholderLabel.Font; } set { _placeholderLabel.Font = value; } }
+		public UIColor PlaceholderColor { get => _placeholderLabel.TextColor;
```
Comment (Julien-Mialon) [inline src/Xmf2.Commons.iOS/Controls/UITextViewWithPlaceholder.cs]: un petit saut de ligne ?

File: src/Xmf2.Commons.iOS/Controls/UITextViewWithPlaceholder.cs [hunk 4]
```diff
@@ -41,8 +42,12 @@ public override string Text
...
+			set => _placeholderLabel.TextColor = value;
+		}
+		public UIFont PlaceholderFont { get => _placeholderLabel.Font;
```
Comment (Julien-Mialon) [inline src/Xmf2.Commons.iOS/Controls/UITextViewWithPlaceholder.cs]: un petit saut de ligne ?

File: src/Xmf2.Commons/OAuth2/OAuth2RestClient.cs [hunk 5]
```diff
@@ -6,10 +6,9 @@
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 using RestSharp.Portable;
-using Xmf2.Rest.HttpClient.Impl;
-using RestClientExtensions = Xmf2.Rest.HttpClient.RestClientExtensions;
+using RestClientExtensions = Xmf2.Commons.HttpClient.RestClientExtensions;
```
Comment (Julien-Mialon) [inline src/Xmf2.Commons/OAuth2/OAuth2RestClient.cs:9]: does this is really useful ?

File: src/Xmf2.Commons/Services/Licences/NoticesHtmlBuilder.cs [hunk 6]
```diff
@@ -7,10 +7,10 @@ namespace Xmf2.Commons.Services.Licences
 {
     public class NoticesHtmlBuilder
     {
-        private Dictionary<Licence, String> mLicenseTextCache = new Dictionary<Licence, String>();
+        private readonly Dictionary<Licence, string> _mLicenseTextCache = new Dictionary<Licence, string>();
```
Comment (Julien-Mialon) [inline src/Xmf2.Commons/Services/Licences/NoticesHtmlBuilder.cs]: on peut enlever le m et mettre un l lower case ?

File: src/Xmf2.Commons/Services/Licences/NoticesHtmlBuilder.cs [hunk 7]
```diff
@@ -69,7 +69,7 @@ public NoticesHtmlBuilder SetClickableUrl(bool isUrlClickable)
             return this;
         }
 
-        public String Build()
+        public string Build()
         {
             StringBuilder noticesHtmlBuilder = new StringBuilder(500);
```
Comment (Julien-Mialon) [inline src/Xmf2.Commons/Services/Licences/NoticesHtmlBuilder.cs]: il sert à quoi le 500 ?

File: src/Xmf2.Rx.Droid/BaseView/ReactiveLinearLayout.cs [hunk 8]
```diff
@@ -52,20 +52,20 @@ object IViewFor.ViewModel
...
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (activated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_activated).OnNext(Unit.Default));
 		}
 
 		public void Deactivate()
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (deactivated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_deactivated).OnNext(Unit.Default));
 		}
```
Comment (Julien-Mialon) [inline src/Xmf2.Rx.Droid/BaseView/ReactiveLinearLayout.cs:69]: Is this really needed to schedule those on main thread ?

File: src/Xmf2.Rx.Droid/ChipClouds/BaseReactiveChipCloudViewHolder.cs [hunk 9]
```diff
@@ -70,12 +70,12 @@ public override void OnViewDetachedFromWindow()
...
 		public void Activate()
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (activated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_activated).OnNext(Unit.Default));
 		}
 
 		public void Deactivate()
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (deactivated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_deactivated).OnNext(Unit.Default));
```
Comment (Julien-Mialon) [inline src/Xmf2.Rx.Droid/ChipClouds/BaseReactiveChipCloudViewHolder.cs:78]: does main thread is really needed ?

File: src/Xmf2.Rx.Droid/LinearList/BaseReactiveLinearLayoutViewViewHolder.cs [hunk 10]
```diff
@@ -10,20 +10,20 @@ namespace Xmf2.Rx.Droid.LinearList
...
 		public void Activate()
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (activated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_activated).OnNext(Unit.Default));
 		}
 
 		public void Deactivate()
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (deactivated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_deactivated).OnNext(Unit.Default));
```
Comment (Julien-Mialon) [inline src/Xmf2.Rx.Droid/LinearList/BaseReactiveLinearLayoutViewViewHolder.cs:26]: main thread needed ?

File: src/Xmf2.Rx.Droid/ListElement/BaseReactiveRecyclerViewViewHolder.cs [hunk 11]
```diff
@@ -71,12 +71,12 @@ protected virtual object GetItemLongClickParameter()
...
 		public void OnViewAttachedToWindow()
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (activated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_activated).OnNext(Unit.Default));
 		}
 
 		public void OnViewDetachedFromWindow()
 		{
-			RxApp.MainThreadScheduler.Schedule(() => (deactivated).OnNext(Unit.Default));
+			RxApp.MainThreadScheduler.Schedule(() => (_deactivated).OnNext(Unit.Default));
```
Comment (Julien-Mialon) [inline src/Xmf2.Rx.Droid/ListElement/BaseReactiveRecyclerViewViewHolder.cs:79]: main thread needed ?

**Review by mcoutanceau — APPROVED** [review]
Ok. Attention au breaking change BaseViewController.BindControls() devenu SetViewModelBindings().

File: src/Xmf2.Commons/OAuth2/OAuth2RestClient.cs (hunk 5 above)
Comment (VJubert) [reply src/Xmf2.Commons/OAuth2/OAuth2RestClient.cs:9]: Conflicts with RestSharp.Portable

**Pull request closed without merging**